### DISCUSSION TITLE
Filter option "EOL" for custom report

### DISCUSSION THREAD

**Discussion by schnudd31do3** [snipe/snipe-it#15424] [Ideas]
Currently I use the purchase date for filtering of assets to be scrapped, but it is a bit incomprehensible. Assets could have different  different warranty periods and it could be that not all concerned assets were listed. So the better way is th filter by "EOL" (end of useful life). Perhaps you can add this filter to the for custom report.

**Comment by snipe** [comment]
Please open this as a feature request, as it's not really a discussion.
